calcom/cal.com
# fix: Remove double encoding for OG image URLs

## What does this PR do?





## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

1. Login with `admin` account.
2. Go to some event type (e.g., http://localhost:3000/admin/quick-chat)
3. Find `meta og:image` element in `head` and run its URL in a new tab
4. See if it renders well
